PCLConfig.cmake.in

```diff
@@ -94,10 +94,7 @@ macro(find_boost)
     set(BOOST_INCLUDEDIR "@Boost_INCLUDE_DIR@")
   endif()
 
-  include(${CMAKE_CURRENT_LIST_DIR}/Modules/AdditionalBoostVersions.cmake)
-  set(Boost_ADDITIONAL_VERSIONS "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@.@Boost_SUBMINOR_VERSION@" "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@" ${Boost_ADDITIONAL_VERSIONS})
-  
-  find_package(Boost 1.65.0 ${QUIET_} COMPONENTS @PCLCONFIG_AVAILABLE_BOOST_MODULES@)
+  find_package(Boost 1.71.0 ${QUIET_} COMPONENTS @PCLCONFIG_AVAILABLE_BOOST_MODULES@ CONFIG)
```

Comment:
```suggestion
  find_dependency(Boost 1.71.0 COMPONENTS @PCLCONFIG_AVAILABLE_BOOST_MODULES@ CONFIG)
```

Needs `include(CMakeFindDependencyMacro)` somewhere above.

Reply:
Thanks for the suggestion, but that is a bit off-topic in this pull request, and I would like to merge this pull request as soon as possible because PCL's incompatibility with Boost 1.86 makes one of our CI checks fail. Perhaps we can look at `find_dependency` again in the future, in another pull request.